crates/python/qcs_sdk/__init__.pyi

```diff
@@ -356,11 +353,16 @@ class RegisterData:
     @staticmethod
     def from_complex32(inner: Sequence[Sequence[complex]]) -> "RegisterData": ...
 
-
 def reset_logging():
     """
     Reset all caches for logging configuration within this library, allowing the most recent Python-side
     changes to be applied.
 
     See <https://docs.rs/pyo3-log/latest/pyo3_log/> for more information.
-    """
\ No newline at end of file
+    """
+
+def gather_diagnostics() -> str:
+    """
+    Gather and return a summary of diagnostics that describe how the package was built and the environment
+    it's running in. Helpful for diagnosing unexpected behavior.
+    """
```

Comment:
Is this needed for direct python exports? Is it that this file clobbers the inferred type hints?